DOCUMENTATION.md

```diff
@@ -564,6 +565,31 @@ const client = await Client.newHazelcastClient(cfg);
 
 In the following chapters you will learn the description of all options supported by Hazelcast Node.js client.
 
+# 3.1 Client Properties
+
+The following is the list of all client properties in alphabetical order.
+
+> **NOTE: When you want to reconfigure a client property, you need to restart the client.**
```

Comment:
Isn't this the same for all of the config elements? We don't support dynamic updates to config elements (or let's say the behavior, in this case, is undefined), so I would remove this warning

Reply:
Yeah this is taken from properties section of IMDG, but let's remove as you said